views/js/controller/routes.js

```diff
@@ -1,14 +1,27 @@
 define(function(){
     return {
-        'TaoProctoring' : {
+        'TestCenter' : {
```

Comment:
not important but to notice, you can use directly an object without a function exec : 
```js
define({ 
  'TestCenter' : { }
});
```